zerver/actions/message_flags.py

```diff
@@ -224,7 +224,7 @@ def do_clear_mobile_push_notifications_for_ids(
     assert len(user_profile_ids) == 1 or len(message_ids) == 1
 
     messages_by_user = defaultdict(list)
-    notifications_to_update = list(
+    notifications_to_update = (
```

Comment:
Do we need this outer parens? I guess it isn't harmful.

Reply:
Black requires them.